tests/models/test_florence2_sam2.py

```diff
@@ -76,3 +147,34 @@ def test_florence2sam2_invalid_media(shared_model):
 @pytest.fixture(scope="module")
 def shared_model():
     return Florence2SAM2()
+
+
+@pytest.fixture
+def assert_predictions(rle_decode_array):
+    def handler(
```

Comment:
should we move `assert_predictions` to a `conftest.py` file?  so we can reuse this fixture

Reply:
let me add this change in another PR so that I don't need to rerun the CI (takes a long time)